### ISSUE TITLE
6-Node Crypto transfer JRS regression has the INSUFFICIENT_ACCOUT_BALANCE error

### ISSUE THREAD

**Issue by ljianghedera** [hashgraph/hedera-services#794]
**Summary of the defect**
This happens from time to time in both master and v090-rc4 branch.

It occurs sometimes around ~35 minutes in the test. It doesn't happen always.

**How to reproduce (if possible)**

The slack link: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1605754841318900

and the corresponding log file URL: https://s3.console.aws.amazon.com/s3/object/hedera-service-regression-jrs?region=us-east-2&prefix=4N_1C/Basic/20201120-0423-AWS-Daily-Services-Comp-Basic-4N-1C/Misc-Basic-1-11m/node0000-TestClient/hapi-client-combined.log

**Service logs (if applicable)**
```
{ accountAmounts { accountID { accountNum: 66211 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66204 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "$\037\377\344\255\273\3709:?\017\212i\247b,\347_\341F\322\335\034\200!4m\227P\363J\231\216\027i\366\227x>\323S\202\r\366\207\032,es\030P\375`\370\235\004\366\372\350\314I\347\001\004" } sigPair { pubKeyPrefix: "\267" ed25519: "\004\023\a\213\021\017t\201\234P\211\304R\352\263\246\334)\325\230\275Z\311f\350?wO\332\207\035\252\302xA\242\377\022v\347\204d!\371\244\023xQ\3034b={du\331y\001Z`\270\265Q\a" }
2020-11-20 10:33:34.660 INFO   140  HapiTxnOp - 'RunCryptoTransfers' -  submitting HapiCryptoTransfer{node=0.0.3} via body=transactionID { transactionValidStart { seconds: 1605868354 nanos: 7117435 } accountID { accountNum: 66204 } } nodeAccountID { accountNum: 3 } transactionFee: 408126 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 66211 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66204 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\303\252\020G\216\346\305\v+\203;\266\024\210\037\367r\340J\204\336\251P)\nSH\321\002\372\360{o\024^\000\234\237\\\251]\2225\335\210\231\362\f\265\002\022\217\233\036k+\357\373\337\256\367\213\331\r" } sigPair { pubKeyPrefix: "\267" ed25519: "\251-\250_\231\271R}\320\367Ii9\004z{)\3462I\336\311\272\016m\244MQ R7\363\030x\366-\270\371\376\272\255\022L9\316\001\rg\2233\326_\306\304\256\330\214w\311\023>\336r\r" }
2020-11-20 10:33:36.178 WARN   199  HapiSpecOperation - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66211 <- +900000000000, 0.0.66204 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:183) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:141) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$7(RunLoadTest.java:109) ~[SuiteRunner.jar:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-20 10:33:36.174 WARN   199  HapiSpecOperation - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66211 <- +900000000000, 0.0.66204 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:183) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:141) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$7(RunLoadTest.java:109) ~[SuiteRunner.jar:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-20 10:33:36.174 WARN   199  HapiSpecOperation - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66211 <- +900000000000, 0.0.66204 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:183) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:141) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$7(RunLoadTest.java:109) ~[SuiteRunner.jar:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-20 10:33:36.176 WARN   199  HapiSpecOperation - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66211 <- +900000000000, 0.0.66204 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]

...
```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

**Comment by ljianghedera** [comment]
This should be considered as an expected error scenarios as it's possible to exhaust all the initial balance of the sender account during the test period with the TPS rate. Will be closed by https://github.com/swirlds/swirlds-platform-regression/pull/677

**Comment by Neeharika-Sompalli** [comment]
Closed by swirlds/swirlds-platform-regression#677
